torch_ort_inference/docker/Dockerfile.ort-infer-stable-torch1120-onnxruntime1120-openvino-ubuntu20.04

```diff
@@ -0,0 +1,86 @@
+#-------------------------------------------------------------------------
+# Copyright(C) 2022 Intel Corporation.
+# SPDX-License-Identifier: MIT
+#--------------------------------------------------------------------------
+
+ARG OPENVINO_VERSION="2022.1.0"
+ARG OVPT_BRANCH="releases/v0.5"
+
+FROM openvino/ubuntu20_runtime:${OPENVINO_VERSION}
+
+LABEL description="This is the runtime image for OpenVINO™ integration with PyTorch on Ubuntu 20.04 LTS"
```

Comment:
"OpenVINO™ integration with PyTorch" should be "OpenVINO™ integration with Torch-ORT"

Reply:
fixed